mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -169,9 +169,12 @@ def show_new_sample(self) -> None:
         Show the new sample in the view and update the spectrum and
         image view accordingly. Resets the ROIs.
         """
-
-        averaged_image = self.model.get_averaged_image()
+        if self.view.normalisation_enabled():
+            averaged_image = self.model.get_normalized_averaged_image()
+        else:
+            averaged_image = self.model.get_averaged_image()
```

Comment:
This also needs to be done in `handle_range_slide_moved()`

Also redraw will be needed in `handle_enable_normalised()` to that the image changes with the normalise box is ticked or unticked.